### ISSUE TITLE
Problem installing Rust and running cargo on Windows 10 64-bit - wrong directory in path

### ISSUE THREAD

**Issue by Dangthrimble** [rust-lang/rustup#2261]
Today I followed the instructions for installing Rust on my Windows 10 64-bit PC. During the install it created the '.rustup\toolchains\stable-x86_64-pc-windows-msvc' directory then complained it couldn't find '.rustup\toolchains\stable-i686-pc-windows-msvc'. I created the latter and the install completed.

`rustup --version` works fine (rustup 1.21.1) but when I run `cargo --version` I get an error and am told "'cargo.exe' is not installed for the toolchain 'stable-x86_64-pc-windows-msvc'" and to run `rustup component add cargo --toolchain stable-x86_64-pc-windows-msvc`.

When I run that, I get an error and am told "toolchain 'stable-x86_64-pc-windows-msvc' does not support components". If I change the command to `rustup component add cargo --toolchain stable-i686-pc-windows-msvc`, I am informed "component 'cargo' for target 'i686-pc-windows-msvc' is up to date".

### Meta

`rustup --version`:
```
rustup 1.21.1 (7832b2ebe 2019-12-20)
```

**Comment by rbtcollins** [comment]
The directory you created by hand is causing corruption / confusion - it is missing important metadata : remove it. 

Then use `rustup toolchain add stable` to install the stable x86-64 toolchain.

**Comment by Dangthrimble** [comment]
But the install was failing until I created the i686 directory and the x86_64 directory is empty.

**Comment by rbtcollins** [comment]
I didn't catch the cross-bit-width discrepancy there. I suggest removing both variants and adding whichever one you actually want again via `rustup toolchain add`

**Comment by Dangthrimble** [comment]
So, let me start again. I have uninstalled Rust using `rustup self uninstall` and on the "Install Rust" page, clicked on "DOWNLOAD RUSTUP-INIT.EXE (64-BIT)" and saved the file.

I have run the executable which selected the following installation options:
   default host triple: x86_64-pc-windows-msvc
     default toolchain: stable
               profile: default
  modify PATH variable: yes

I proceeded with the default installation and everything worked perfectly!

So I don't know what happened, but everything looks good now. Apologies.
